<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-09-14T23:03:37 IST" name="Default suite" finished-at="2023-09-14T23:04:42 IST" duration-ms="64513">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.dBankdemo.Testscripts.TestRunner@6f10d5b6]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2023-09-14T23:03:37 IST" name="Default test" finished-at="2023-09-14T23:04:42 IST" duration-ms="64513">
      <class name="com.dBankdemo.Testscripts.TestRunner">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:com.dBankdemo.Testscripts.TestRunner@6f10d5b6]" started-at="2023-09-14T23:03:37 IST" name="setUpClass" finished-at="2023-09-14T23:03:38 IST" duration-ms="614" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@5384ce66]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.dBankdemo.Testscripts.TestRunner@6f10d5b6]" started-at="2023-09-14T23:03:38 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2023-09-14T23:04:39 IST" duration-ms="60982" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Login with valid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Withdraw money from an account]"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Account Number: 486171182] but found [Account Number: 486171185]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Account Number: 486171182] but found [Account Number: 486171185]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.dBankdemo.Testscripts.WithdrawMoney.the_user_should_verify(WithdrawMoney.java:65)
at âœ½.the User should Verify-"Account Number: 486171182"(file:///C:/Users/Hello/eclipse-workspace/RllDigitalBank/Feature/withdraw_money_from_account.feature:7)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:com.dBankdemo.Testscripts.TestRunner@6f10d5b6]" started-at="2023-09-14T23:04:39 IST" name="tearDownClass" finished-at="2023-09-14T23:04:42 IST" duration-ms="2866" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- com.dBankdemo.Testscripts.TestRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
